src/style.css

```
@@ -1 +1,64 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  display: flex;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  align-items: center;
+  text-transform: uppercase;
+}
+
+.logo__Moyo {
+  width: 40px;
+  height: 40px;
+}
+
+.menu__Item {
+  position: relative;
+}
+
+.menu__Item:not(:last-child) {
+  padding-right: 20px;
+}
+
+.menu__link {
+  display: block;
+  padding: 23px 0;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
+}
+
+ul {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.menu__Item a:hover {
```

```suggestion
menuLink:hover {
```
У тебя есть класс для этого элемента